### PR TITLE
Add Travis HOWTO

### PR DESCRIPTION
**Pull request by pepijn**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,6 +35,16 @@ High level guidelines:
 
 * Coming soon
 
+## Travis CI
+
+Adding Travis CI to your repository:
+
+1. Visit your profile page: https://magnum.travis-ci.com/profile
+2. Go to the Blendle account in the sidebar
+3. Flip the ON/OFF switch for you project
```
Comment (koenrh) [inline README.md]: Typo: "for youR project"

Comment (pepijn) [inline README.md]: ty, fixed :smile_cat:

**Comment by koenrh** [comment]
:+1:

**Comment by eterps** [comment]
Thanks :100:

**Pull request merged**